frontend/src/bundles/common/components/stepper/common/components/select-icon/select-icon.tsx

```diff
@@ -0,0 +1,62 @@
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
+import { IconMap, Size } from '~/bundles/common/icons/icons.js';
+
+import { Icon, Text } from '../../../../components.js';
+
+const selectIcon = (
```

Comment:
I assumed that it is component because it is in components folder. It would be better to either replace this function to other dir or rewrite it as components and leave it here.